From 30a8836090551ac1271b156bfdf40f9d0f4e6f3e Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@freescale.com>
Date: Thu, 26 Nov 2015 10:48:05 +0800
Subject: [PATCH 0681/5242] MLK-11896-1 usb: chipidea: imx: add break for
 switch case

commit  d84837a485f4e43be24e7295b9f6cf8af43a2420 from
https://source.codeaurora.org/external/imx/linux-imx.git

Fix the coverity check complains although there is no function issue
with current code.

Signed-off-by: Li Jun <jun.li@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/usb/chipidea/ci_hdrc_imx.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/usb/chipidea/ci_hdrc_imx.c b/drivers/usb/chipidea/ci_hdrc_imx.c
index 3973a16..4ad4a5f 100644
--- a/drivers/usb/chipidea/ci_hdrc_imx.c
+++ b/drivers/usb/chipidea/ci_hdrc_imx.c
@@ -365,10 +365,12 @@ static int ci_hdrc_imx_notify_event(struct ci_hdrc *ci, unsigned event)
 		if (data->usbmisc_data)
 			return imx_usbmisc_term_select_override(
 					data->usbmisc_data, true, 1);
+		break;
 	case CI_HDRC_IMX_TERM_SELECT_OVERRIDE_OFF:
 		if (data->usbmisc_data)
 			return imx_usbmisc_term_select_override(
 					data->usbmisc_data, false, 0);
+		break;
 	default:
 		dev_dbg(dev, "unknown event\n");
 	}
-- 
1.7.9.5

